### PR TITLE
Payment flow: basic HTML structure

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## Why are you doing this?

This is enough to get me started with the CSS. This HTML can already be tested for a11y if necessary. It is also a good opportunity to make sure all the content that will be displayed in this page is present, before layers of style and interaction mess it all up.

Currently missing afaik:
- [ ] SVG for the circle logo (cc @Amohkhan )
- [ ] error messages
- [ ] success message

[**Trello Card**](https://trello.com)

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
(Breaking because [this one](#843) needs to go first)

File: app/views/newPaymentFlow.scala.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    <input id="contributionLName" type="text" required>
+  </div>
+
+  <div>
+    <label for="contributionEmail">Email address</label>
+    <input id="contributionEmail" type="email" required>
+  </div>
+
+  <div>
+    <label for="contributionState">Select your state</label>
```
Comment (regiskuckaertz) [inline app/views/newPaymentFlow.scala.html]: Weird that this is the only label using a verb (cc @ionamckendrick )

**Comment by regiskuckaertz** [comment]
For reviewers: this is for you to focus on the semantics. Things may change in the future, i.e. I may add attributes and meaningless elements (`DIV` or `SPAN`) but no other semantic element.

It is too early to say whether the whole decorum (metadata, banners and footers) will fit with the rest of the site so this can probably wait until we have a better understanding of how this redesign will affect other pages (if it does).

**Comment by regiskuckaertz** [comment]
@Ap0c thanks a lot!

**Comment by regiskuckaertz** [comment]
Will provide the remaining bits as soon as I get the material from design, in a separate PR.

File: app/views/newPaymentFlow.scala.html (hunk 1 above)
Comment (ionamckendrick) [reply app/views/newPaymentFlow.scala.html]: I agree, actually. This was a hangover from what we currently have. I'd suggest changing it to "State" moving forward. Good spot!

**Pull request merged**